Update to coding standards.

fklein-lu opened the pull request.

boonebgorges commented:
@fklein-lu Hi there - any reason why you deleted your branch? I'd be glad to merge.

fklein-lu commented:
@boonebgorges Hey, deleted the PR because I meant to PR against the HM fork, but my click finger was too quick. 😁 

I can reopen the PR, but I did this super quick, and super dirty, so it isn't exactly my finest work. Basically ran the code through HM's coding standards and fixed whatever came up.

Thanks for the plugin, works great! 👍

boonebgorges commented:
Ha ha, I also have a too-quick clicker-finger.

A rough cleanup may be better than nothing, if you don't want to sink any more time into it. Either way, don't be shy about sending a PR my way at some point, and we'll get it merged upstream so you don't have to maintain a fork.
